Uplift/Views/GymDetail/Facilities/CourtView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CourtView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/3/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CourtView: UICollectionViewCell {
```

Comment:
because this is the cell that for the `Gymnasium` section of Facilities, lets call this `GymnasiumCollectionViewCell`